.rubocop_todo.yml

```diff
@@ -89,7 +89,7 @@ Metrics/MethodLength:
 # Offense count: 21
 # Configuration parameters: CountComments.
 Metrics/ModuleLength:
-  Max: 408
+  Max: 431
```

Comment:
Can you find some stuff to split out in the helper so that it doesn't go beyond the limit? https://github.com/theforeman/foreman/pull/3621/files I had to do that here , if that works for you feel free to take it